tripal_chado/config/install/tripal.tripalfield_collection.general_chado.yml

```diff
@@ -235,6 +234,58 @@ fields:
               region: content
               weight: 45
 
+    -   name: organism_image
+        content_type: organism
+        label: Organism Image
+        type: image
+        description: One or more images for the organism.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: drupal_sql_storage
+            storage_plugin_settings: 
+                base_table: organism
+        settings:
+            termIdSpace: IAO
+            termAccession: "0000101"
```

Comment:
Just a comment, I have been thinking for a while about creating a field for the `chado.eimage` table. On CarrotOmics I have a Tripal 3 field to do this and use this very term for the field. Would it be a conflict to use the same term as this field uses?